ansible/roles/uan_interfaces/tasks/access_control.yml

```diff
@@ -70,7 +70,7 @@
       - bash -c "iptables -A OUTPUT -d {{customer_access_network}} -j DROP"
     when: customer_access_network is defined
 
-  when: uan_can_setup
+  when: (uan_user_access_cfg | upper) == "CAN"
```

Comment:
Do we need to enable this feature for the CHN?

Reply:
I will do that!  Good catch!

Reply:
Fixed!